UnderSea.Angular/src/app/core/pages/login/login.component.ts

```diff
@@ -46,32 +46,33 @@ export class LoginComponent implements OnInit {
   }
 
   login(): void {
-
-    this.authService.login(this.username, this.password).pipe(
-      tap(tokens => {
+    this.authService.login(
+      this.loginForm.controls.username.value,
+      this.loginForm.controls.password.value)
+      .subscribe(tokens => {
         if (tokens.accessToken != null) {
           this.router.navigate(['/main']);
         }
-      })
-    ).subscribe();
+      });
   }
 
   signup(): void {
     this.authService.signup(
       this.regForm.controls.regUsername.value,
       this.regForm.controls.regPassword.value,
       this.regForm.controls.country.value).pipe(
-      tap(res => {
+        tap(res => {
+          this.snackbar.open('Sikeres regisztráció', 'Bezár', {
+            duration: 3000,
+            panelClass: ['my-snackbar'],
+          });
+        })
+      ).subscribe(res => {
```

Comment:
ez nem működött?

Reply:
Működött szerintem, nem tudom pontosan miért raktam át eredetileg. Most visszaírtam a tap-be